frontend/src/posapp/components/pos/Payments.vue

```
@@ -1098,6 +1118,28 @@ watch(
 );
 
+watch(
+	invoiceType,
+	(data) => {
+		if (invoice_doc.value && data !== "Order") {
+			invoice_doc.value.posa_delivery_date = null;
+			invoice_doc.value.posa_notes = null;
+			invoice_doc.value.posa_authorization_code = null;
+			invoice_doc.value.shipping_address_name = null;
+		} else if (invoice_doc.value && data === "Order") {
+			new_delivery_date.value = formatDateDisplay(frappe.datetime.now_date());
```

ESLint

[error] 'frappe' is not defined.

(no-undef)